.github/workflows/docs.yaml

```diff
@@ -35,21 +35,23 @@ jobs:
       - name: Build docs
         run: |
           make docs
+          cp docs/index.html index.html
 
       - name: Prepare gh-pages branch
         run: |
-          git restore docs/assemblies/custom_resources.adoc
           git config user.name github-actions
           git config user.email github-actions@github.com
 
           git branch -D gh-pages &>/dev/null || true
           git checkout -b gh-pages 4cd0193fc6c5bc7e76f3a0148d0447fb0d7fbe6a
+          git revert 4cd0193fc6c5bc7e76f3a0148d0447fb0d7fbe6a
```

Comment:
Please add a comment about why are we reverting this.

Reply:
it is just for creating an empty branch, but I found a better way

Reply:
++ for `--orphan` usage